cylc/flow/workflow_files.py

```diff
@@ -603,7 +604,9 @@ def _clean_check(reg, run_dir):
             f"Cannot remove running workflow.\n\n{exc}")
 
 
-def init_clean(reg: str, opts: 'Values') -> None:
+def init_clean(
+    reg: str, opts: 'Values', badhosts: Optional[Set[str]] = None
+) -> None:
```

Comment:
`init_clean()` is only called from `cylc.flow.scripts.clean`, so I'm not sure where `badhosts` comes into it?

Reply:
No, I think that you are right. This is effectively a standalone app? I believe that I have now fixed it.

Reply:
I'm not seeing any changes

Reply:
Change made, but not pushed.